### ISSUE TITLE
local server error

### ISSUE THREAD

**Issue by janeinju** [Acly/krita-ai-diffusion#1281]
As shown in the screenshot, the local server is not connected, and the model checkpoint is not recognized in the style (there is a model file in the folder)
![1](https://github.com/user-attachments/assets/271fcc03-5462-4572-8477-9996fc6adbc5)
![2](https://github.com/user-attachments/assets/06f986db-fd78-4071-80a1-7711ac058a45)
How can I solve this? I'd like to ask for help.
ps, Environment: Windows 10 64 bit

**Comment by Acly** [comment]
Please check the logs (client.log) regarding the error message.

Listing checkpoints only works once server is running and connected.

**Comment by janeinju** [comment]
When you first install it, it runs automatically, but after a few days,
this error occurs when you run it. For reference, I know RTX 3080.

2024년 10월 19일 (토) 오전 5:38, Acly ***@***.***>님이 작성:

**Comment by janeinju** [comment]
Could this be related to stable diffusion? Could this error occur if stable
diffusion is not run properly?

2024년 10월 19일 (토) 오전 9:05, inju jeon ***@***.***>님이 작성:

**Comment by janeinju** [comment]
The client contents appear like this, but I don't know what to do.
(2024-10-19 10:48:08,091 INFO Loaded 11 sampler presets from C:\Users\jeoninju\AppData\Roaming\krita\pykrita\ai_diffusion\presets\samplers.json
2024-10-19 10:48:08,135 INFO Extension initialized, Version: 1.25.0, Python: 3.10.7 (tags/v3.10.7:6cc6b13, Sep  5 2022, 14:08:36) [MSC v.1933 64 bit (AMD64)]
2024-10-19 10:48:08,136 INFO Loading settings from C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\settings.json
2024-10-19 10:48:08,137 INFO Found server installation v1.25.0 at C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server
2024-10-19 10:48:08,154 INFO Loaded 8 model files from C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\database\loras.json
2024-10-19 10:48:08,495 INFO Checking for latest plugin version at https://api.interstice.cloud
2024-10-19 10:48:10,100 INFO Starting server with python -su -Xutf8 main.py
2024-10-19 10:48:10,412 ERROR Failed to attach process to job: AssignProcessToJobObject failed: 5
2024-10-19 10:48:10,475 INFO New plugin version available: 1.26.0
2024-10-19 10:48:10,662 WARNING Failed to launch/connect server at startup: Error during server startup:  [0]
2024-10-19 10:48:27,127 INFO Downloading plugin update https://files.interstice.cloud/plugin/krita_ai_diffusion-1.26.0.zip
2024-10-19 10:48:28,471 INFO Extracting plugin archive into C:\Users\jeoninju\AppData\Local\Temp\tmp_2nzdvnv\krita_ai_diffusion-1.26.0
2024-10-19 10:48:28,721 INFO Installing new plugin version to C:\Users\jeoninju\AppData\Roaming\krita\pykrita
2024-10-19 10:49:25,539 INFO Loaded 11 sampler presets from C:\Users\jeoninju\AppData\Roaming\krita\pykrita\ai_diffusion\presets\samplers.json
2024-10-19 10:49:25,618 INFO Extension initialized, Version: 1.26.0, Python: 3.10.7 (tags/v3.10.7:6cc6b13, Sep  5 2022, 14:08:36) [MSC v.1933 64 bit (AMD64)]
2024-10-19 10:49:25,618 INFO Loading settings from C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\settings.json
2024-10-19 10:49:25,619 INFO Found server installation v1.25.0 at C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server
2024-10-19 10:49:25,623 INFO Loaded 8 model files from C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\database\loras.json
2024-10-19 10:49:25,792 INFO Checking for latest plugin version at https://api.interstice.cloud
2024-10-19 10:49:27,495 INFO Plugin is up to date!
2024-10-19 10:49:46,061 INFO Starting upgrade from 1.25.0 to 1.26.0
2024-10-19 10:49:46,061 INFO Backing up C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\ComfyUI to C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\upgrade-1.26.0\ComfyUI
2024-10-19 10:49:46,061 INFO Installation started in C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server
2024-10-19 10:49:46,333 INFO Using Python: , C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\python\python.exe
2024-10-19 10:49:46,585 INFO Using pip: 
2024-10-19 10:49:46,586 INFO Downloading https://github.com/comfyanonymous/ComfyUI/archive/1b8089528502a881d0ed2918b2abd54441743dd0.zip to C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\.cache\ComfyUI-1b8089528502a881d0ed2918b2abd54441743dd0.zip
2024-10-19 10:49:47,367 INFO Extracting C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\.cache\ComfyUI-1b8089528502a881d0ed2918b2abd54441743dd0.zip to C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server
2024-10-19 10:49:47,570 INFO Executing C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\python\python.exe -su -m pip install torch~=2.4.1 torchvision~=0.19.1 torchaudio~=2.4.1 --index-url https://download.pytorch.org/whl/cu124
2024-10-19 10:49:47,754 ERROR Failed to attach process to job: AssignProcessToJobObject failed: 5
2024-10-19 10:49:47,865 INFO Executing C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\python\python.exe -su -m pip install -r C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\ComfyUI-1b8089528502a881d0ed2918b2abd54441743dd0\requirements.txt
2024-10-19 10:49:48,066 ERROR Failed to attach process to job: AssignProcessToJobObject failed: 5
2024-10-19 10:49:48,180 INFO Finished installing ComfyUI
2024-10-19 10:49:48,181 INFO Using C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\.cache\comfyui_controlnet_aux-6c563c5032f77dd3336b603bde3d12b415d003ab.zip
2024-10-19 10:49:48,181 INFO Extracting C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\.cache\comfyui_controlnet_aux-6c563c5032f77dd3336b603bde3d12b415d003ab.zip to C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\ComfyUI\custom_nodes
2024-10-19 10:49:49,038 INFO Executing C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\python\python.exe -su -m pip install -r C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\ComfyUI\custom_nodes\comfyui_controlnet_aux\requirements.txt
2024-10-19 10:49:49,212 ERROR Failed to attach process to job: AssignProcessToJobObject failed: 5
2024-10-19 10:49:49,336 INFO Finished installing ControlNet Preprocessors
2024-10-19 10:49:49,336 INFO Using C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\.cache\ComfyUI_IPAdapter_plus-88a71407c545e4eb0f223294f5b56302ef8696f3.zip
2024-10-19 10:49:49,337 INFO Extracting C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\.cache\ComfyUI_IPAdapter_plus-88a71407c545e4eb0f223294f5b56302ef8696f3.zip to C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\ComfyUI\custom_nodes
2024-10-19 10:49:49,357 INFO Finished installing IP-Adapter
2024-10-19 10:49:49,358 INFO Downloading https://github.com/Acly/comfyui-tooling-nodes/archive/ef5ccfa98f035fd1d9fd00a21ac4ab75a5f73061.zip to C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\.cache\comfyui-tooling-nodes-ef5ccfa98f035fd1d9fd00a21ac4ab75a5f73061.zip
2024-10-19 10:49:50,985 INFO Extracting C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\.cache\comfyui-tooling-nodes-ef5ccfa98f035fd1d9fd00a21ac4ab75a5f73061.zip to C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\ComfyUI\custom_nodes
2024-10-19 10:49:51,001 INFO Executing C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\python\python.exe -su -m pip install -r C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\ComfyUI\custom_nodes\comfyui-tooling-nodes\requirements.txt
2024-10-19 10:49:51,162 ERROR Failed to attach process to job: AssignProcessToJobObject failed: 5
2024-10-19 10:49:51,290 INFO Finished installing External Tooling Nodes
2024-10-19 10:49:51,291 INFO Using C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\.cache\comfyui-inpaint-nodes-6ce66ff1b5ed4e5819b23ccf1feb976ef479528a.zip
2024-10-19 10:49:51,291 INFO Extracting C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\.cache\comfyui-inpaint-nodes-6ce66ff1b5ed4e5819b23ccf1feb976ef479528a.zip to C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\ComfyUI\custom_nodes
2024-10-19 10:49:51,319 INFO Finished installing Inpaint Nodes
2024-10-19 10:49:51,320 INFO Using C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\.cache\ComfyUI-GGUF-454955ead3336322215a206edbd7191eb130bba0.zip
2024-10-19 10:49:51,320 INFO Extracting C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\.cache\ComfyUI-GGUF-454955ead3336322215a206edbd7191eb130bba0.zip to C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\ComfyUI\custom_nodes
2024-10-19 10:49:51,328 INFO Executing C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\python\python.exe -su -m pip install -r C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\ComfyUI\custom_nodes\ComfyUI-GGUF\requirements.txt
2024-10-19 10:49:51,514 ERROR Failed to attach process to job: AssignProcessToJobObject failed: 5
2024-10-19 10:49:51,637 INFO Finished installing GGUF
2024-10-19 10:49:51,638 INFO Installation finished in C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server
2024-10-19 10:49:51,638 INFO Found server installation v1.26.0 at C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server
2024-10-19 10:49:51,643 INFO Migrating C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\ComfyUI\models
2024-10-19 10:49:51,657 INFO Migrating C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\ComfyUI\custom_nodes\comfyui_controlnet_aux\ckpts
2024-10-19 10:49:51,658 INFO Found server installation v1.26.0 at C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server
2024-10-19 10:49:52,705 INFO Finished upgrade to 1.26.0
2024-10-19 10:49:52,711 INFO Starting server with python -su -Xutf8 main.py
2024-10-19 10:49:52,862 ERROR Failed to attach process to job: AssignProcessToJobObject failed: 5
2024-10-19 10:49:59,798 INFO Starting server with python -su -Xutf8 main.py
2024-10-19 10:49:59,923 ERROR Failed to attach process to job: AssignProcessToJobObject failed: 5
2024-10-19 10:50:11,802 INFO Starting server with python -su -Xutf8 main.py
2024-10-19 10:50:11,943 ERROR Failed to attach process to job: AssignProcessToJobObject failed: 5
2024-10-19 10:50:13,099 INFO Starting server with python -su -Xutf8 main.py
2024-10-19 10:50:13,224 ERROR Failed to attach process to job: AssignProcessToJobObject failed: 5
2024-10-19 10:51:33,672 INFO Loaded 11 sampler presets from C:\Users\jeoninju\AppData\Roaming\krita\pykrita\ai_diffusion\presets\samplers.json
2024-10-19 10:51:33,696 INFO Extension initialized, Version: 1.26.0, Python: 3.10.7 (tags/v3.10.7:6cc6b13, Sep  5 2022, 14:08:36) [MSC v.1933 64 bit (AMD64)]
2024-10-19 10:51:33,696 INFO Loading settings from C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\settings.json
2024-10-19 10:51:33,697 INFO Found server installation v1.26.0 at C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server
2024-10-19 10:51:33,701 INFO Loaded 8 model files from C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\database\loras.json
2024-10-19 10:51:33,866 INFO Checking for latest plugin version at https://api.interstice.cloud
2024-10-19 10:51:35,314 INFO Starting server with python -su -Xutf8 main.py
2024-10-19 10:51:35,517 ERROR Failed to attach process to job: AssignProcessToJobObject failed: 5
2024-10-19 10:51:35,768 WARNING Failed to launch/connect server at startup: Error during server startup:  [0]
2024-10-19 10:51:35,924 INFO Plugin is up to date!
2024-10-19 10:51:44,277 INFO Starting server with python -su -Xutf8 main.py
2024-10-19 10:51:44,402 ERROR Failed to attach process to job: AssignProcessToJobObject failed: 5
2024-10-19 10:51:45,964 INFO Starting server with python -su -Xutf8 main.py
2024-10-19 10:51:46,089 ERROR Failed to attach process to job: AssignProcessToJobObject failed: 5
2024-10-19 10:51:46,558 INFO Starting server with python -su -Xutf8 main.py
2024-10-19 10:51:46,682 ERROR Failed to attach process to job: AssignProcessToJobObject failed: 5
2024-10-19 10:53:06,748 INFO Loaded 11 sampler presets from C:\Users\jeoninju\AppData\Roaming\krita\pykrita\ai_diffusion\presets\samplers.json
2024-10-19 10:53:06,771 INFO Extension initialized, Version: 1.26.0, Python: 3.10.7 (tags/v3.10.7:6cc6b13, Sep  5 2022, 14:08:36) [MSC v.1933 64 bit (AMD64)]
2024-10-19 10:53:06,771 INFO Loading settings from C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\settings.json
2024-10-19 10:53:06,772 INFO Found server installation v1.26.0 at C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server
2024-10-19 10:53:06,776 INFO Loaded 8 model files from C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\database\loras.json
2024-10-19 10:53:06,941 INFO Checking for latest plugin version at https://api.interstice.cloud
2024-10-19 10:53:08,418 INFO Starting server with python -su -Xutf8 main.py
2024-10-19 10:53:08,628 ERROR Failed to attach process to job: AssignProcessToJobObject failed: 5
2024-10-19 10:53:08,878 WARNING Failed to launch/connect server at startup: Error during server startup:  [0]
2024-10-19 10:53:09,003 INFO Plugin is up to date!
2024-10-19 10:53:27,488 INFO Starting server with python -su -Xutf8 main.py
2024-10-19 10:53:27,613 ERROR Failed to attach process to job: AssignProcessToJobObject failed: 5)

**Comment by Acly** [comment]
It looks like the server process immediately shuts down after it was started, without reporting any kind of error. Hard to tell what caused this.

You could try to start it separately, maybe it will tell more. Open a Command Prompt and run
```
C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\python\python.exe -su C:\Users\jeoninju\AppData\Roaming\krita\ai_diffusion\server\ComfyUI\main.py
```
